### DISCUSSION TITLE
部署后的文章列表中，文章的时间与预期不符

### DISCUSSION THREAD

**Discussion by guowei-gong** [YunYouJun/hexo-theme-starry#344] [General 大局为重]
在我的本地，文章创建时间与更新时间看起来是正常的，如下。
![image](https://user-images.githubusercontent.com/62499904/186572714-c0751312-e8be-4a6d-ae05-49a9c3d4501c.png)

当我部署到 `github.io` 后，文章时间相关的信息变得与预期不符！如下。
![image](https://user-images.githubusercontent.com/62499904/186572655-08e2bdef-8b71-4b4f-b0c7-e7856d014f8a.png)

请问我应该如何保证文章时间相关信息，本地与服务器一致？
或者，我如何隐藏更新时间，保留创建时间？🤔

**Comment by YunYouJun** [comment]
应该是在部署时 Git 信息发生了变动。

建议始终手动标记更新时间，譬如：

```bash
---
date: 2021-01-01
updated: 2022-08-30
---
```

**Reply by guowei-gong** [reply]
tks
